[IMP] l10n_pe: eCommerce electronic invoicing (#10495)
odoo/documentation: content/applications/finance/fiscal_localizations/peru.rst

   defined *Quantity*, *Price* and *Tax*.

.. note::
   If you will be offering free delivery, then you would have to manually remove the
   :guilabel:`Delivery Product`, or at least use `$0.01` (one cent) for the invoice to be validated
   with SUNAT.

   .. image:: peru/l10npe-free-shipping-invoicing-pe-ecommerce.png
      :alt: Free shipping configuration for Peruvian eCommerce.

Invoicing flow for eCommerce
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Once the :ref:`configurations <peru/ecommerce-configuration>` are all set, the following sections
cover the invoicing flow.

Client account creation
***********************

If the client does not have an account on your website already, they can create a new one from the
:menuselection:`Website` by clicking :guilabel:`Sign In` and then :guilabel:`Don't have an account?`
button.

If you want to manage the clients' access in the backend, review the
:doc:`../../general/users/portal` documentation.

Input fiscal fields in the checkout process
*******************************************

As mentioned above, fiscal field inputs are available in the checkout process. Entering fiscal data

> enables the purchase to conclude in the corresponding electronic document.

Reviewer: ```suggestion
enables the purchase to create the corresponding electronic document.
```

When the client makes a successful purchase and payment, the invoice is generated with the
corresponding |EDI| elements, and the document type (Boleta/Factura) is selected based on the tax ID
defined by the contact (RUC/DNI).

Even though the invoice is created, it needs to :ref:`be sent to the OSE and the SUNAT
<peru/electronic-invoice-status>`; that can be manually done per invoice, or the default scheduled
action sends all published invoices once a day.

> Once the invoice is validated with |SUNAT|, the user can download the :file:`.zip` with the CDR, XML
> and PDF directly from the portal view.

Reviewer: ```suggestion
Once the invoice is validated with |SUNAT|, the user can download the :file:`.zip` with the CDR,
XML, and PDF directly from the portal view by clicking the :guilabel:`Download` button.
```

Reviewer: We're in the process of publishing new documentation guidelines, but one of the issues that we're implementing is to remove any markups from images. Also, we want the written documentation to be clear enough to stand on its own without screenshots, which it wasn't originally, but with the minor addition telling the user where to download the file, it should be clear enough to remove the screenshot.

> 
> .. image:: peru/l10npe-edi-files-pe-ecommerce.png
>    :alt: Downloading electronic invoicing files from portal view.

Reviewer: ```suggestion
```
